prov/shm/src/smr.h

```diff
@@ -448,5 +448,68 @@ static inline int smr_cma_loop(pid_t pid, struct iovec *local,
 	}
 }
 
+static inline struct smr_cmd_entry *
+smr_get_cmd(struct smr_region *smr, int64_t *pos)
+{
+	struct smr_cmd_entry *ce = NULL;
+	int ret;
+
+	ret = smr_cmd_queue_next(smr_cmd_queue(smr), &ce, pos);
+
+	if (ret == -FI_ENOENT)
+		return NULL;
+
+	return ce;
+}
+
+static inline void smr_queue_cmd(struct smr_cmd_entry *ce,
+				 int64_t pos)
+{
+	smr_cmd_queue_commit(ce, pos);
+}
+
+static inline struct smr_cmd_entry *
+smr_recv_cmd(struct smr_region *smr, int64_t *pos)
+{
+	struct smr_cmd_entry *ce = NULL;
+	int ret;
+
+	ret = smr_cmd_queue_head(smr_cmd_queue(smr), &ce, pos);
+
+	if (ret == -FI_ENOENT)
+		return NULL;
+
+	return ce;
+}
+
+static inline void
+smr_discard_cmd(struct smr_region *smr,
+		struct smr_cmd_entry *ce,
+		int64_t pos)
+{
+	smr_cmd_queue_discard(smr_cmd_queue(smr), ce, pos);
+}
```

Comment:
See above comment - don't have single-line functions that are straight pass-thru